load-test-framework/src/main/java/com/google/pubsub/flic/Driver.java

```diff
@@ -142,6 +156,41 @@
   )
   private int numberOfMessages = 0;
   @Parameter(
+      names = {"--max_publish_latency_test"},
+      description = "In this test we will continuously run load tests with increasing request " +
+          "rates until we hit max_publish_latency_millis. You must only provide a single type of " +
+          "publisher to use this test. This uses the latency specified by " +
+          "max_publish_latency_percentile as the bound to check."
+  )
+  private boolean maxPublishLatencyTest = false;
+  @Parameter(
+      names = {"--max_publish_latency_millis"},
+      description = "This sets the maximum latency in milliseconds, in this test we will" +
```

Comment:
This should just say "This is the maximum latency in milliseconds allowed before terminating the max_publish_latency_test.